src/torch/0-tiefvision-commons/io/tiefvision_redis_io.lua

```diff
@@ -15,46 +15,36 @@ local redis = require 'redis'
 local redisClient = nil
 
 local tiefvision_redis_io = {}
-function tiefvision_redis_io.read(fileName)
-  local key = toKey(fileName)
-  local response = tiefvision_redis_io.redisClient:hgetall(key)
-
-  local responseInt = {}
-  for key, value in pairs(response) do
-    local keyInt = tonumber(key)
-    responseInt[keyInt] = value
+function tiefvision_redis_io.read(database, key)
+  local response = tiefvision_redis_io.redisClient:hgetall(database .. ':' .. key)
```

Comment:
This is just for my own curiosity, but why are we overwriting the pairs in `response`. I mean, if that's the response we got from Redis, and we don't seem to be doing anything with the values `v` in the loop below, what are we actually doing by overwriting each key `k`?